libminifi/include/utils/file/PathUtils.h

```diff
@@ -17,12 +17,20 @@
 #ifndef LIBMINIFI_INCLUDE_UTILS_FILE_PATHUTILS_H_
 #define LIBMINIFI_INCLUDE_UTILS_FILE_PATHUTILS_H_
 
+#include <climits>
 #include <cctype>
 #include <cinttypes>
 #include <memory>
 #include <string>
 #include <system_error>
 #include <utility>
+#include "utils/OptionalUtils.h"
+
+#ifdef _MSC_VER
```

Comment:
not clear to me, it was moved from `FlowController.cpp` (and defined the same way in `MainHelper.h`)

Reply:
I think what's going on is that the constant is `PATH_MAX` on Linux but `MAX_PATH` on Windows.  I don't know why (or if) `canonicalize()` works on Windows now, but it would probably be better to change the constant on line 70 to `MAX_PATH`.

Reply:
I removed this macro altogether since it wasn't even needed (`PATH_MAX` is used here exactly on non-win environments)

Reply:
ah yes, I missed the "n" of `#ifndef` -- never mind